espnet/nets/chainer_backend/ctc.py

```diff
@@ -127,6 +129,35 @@ def __call__(self, hs, ys):
 
         return self.loss
 
+    def forward_from_transformer(self, hs, ys):
+        """Core function of the Warp-CTC layer.
+
+        Args:
+            hs (iterable of chainer.Variable | N-dimention array): Input variable from encoder.
+            ys (iterable of chainer.Variable | N-dimension array): Input variable of decoder.
```

Comment:
I do not think the `ys` can be chainer.Variable. see https://github.com/jheymann85/chainer_ctc/blob/master/chainer_ctc/warpctc.py#L34